crt_portal/cts_forms/models.py

```diff
@@ -544,6 +544,10 @@ def status_assignee_reset(self):
         self.assigned_to = None
         self.status = 'new'
         self.primary_statute = None
+        self.retention_schedule = None
```

Comment:
would you mind changing the name of this function away from `status_assignee_reset`? Maybe `reset_for_changed_section` or something?